plugins/poll/assets/javascripts/lib/chart-colors.js.es6

```diff
@@ -0,0 +1,71 @@
+export function getColors(count, palette) {
+  palette = palette || "cool";
+  let gradient;
+
+  switch (palette) {
```

Comment:
One issue here is these colours are not really customizable via theme. That might be OK but it's something to think about.